css/css-fonts/parsing/font-face-src-tech.html

```diff
@@ -15,23 +15,19 @@
     // Check that each valid keyword is accepted
     { src: 'url("foo.ttf") tech(feature-opentype)', valid: true },
     { src: 'url("foo.ttf") tech(feature-aat)', valid: true },
-    { src: 'url("foo.ttf") tech(feature-graphite)', valid: true },
-    { src: 'url("foo.ttf") tech(color-colrv0)', valid: true },
-    { src: 'url("foo.ttf") tech(color-colrv1)', valid: true },
-    { src: 'url("foo.ttf") tech(color-svg)', valid: true },
+    { src: 'url("foo.ttf") tech(color-COLRv0)', valid: true },
+    { src: 'url("foo.ttf") tech(color-COLRv1)', valid: true },
     { src: 'url("foo.ttf") tech(color-sbix)', valid: true },
-    { src: 'url("foo.ttf") tech(color-cbdt)', valid: true },
+    { src: 'url("foo.ttf") tech(color-CBDT)', valid: true },
```

Comment:
These are ascii-case-insensitive, why change the case? Is it just for readability?

Reply:
Because they are compared against the serialization, which may return the abbreviation part in uppercase. Alternatively, we can lowercase everything below in the comparison step.